.azure-pipelines/templates/run-tests.yml

```diff
@@ -27,3 +27,5 @@ steps:
   inputs:
     testResultsFiles: '**/test-*.xml'
     testRunTitle: '${{ parameters.test_run }}'
+    mergeTestResults: true
+    failTaskOnFailedTests: true
```

Comment:
Need this?

Reply:
@ofek Are you referring to `failTaskOnFailedTests` ? 

No necessarily needed, but I think setting `failTaskOnFailedTests: true` make sense. What do you think ?

> When selected, the task will fail if any of the tests in the results file is marked as failed. The default is false, which will simply publish the results from the results file.

Reply:
Removed. 

Actually, there is no much value having that. It will just make one more `step` to fail. But the whole job is failing anyway, so, no value and makes the UI more confusing. So removed.

![image](https://user-images.githubusercontent.com/49917914/67618463-4dbdb200-f7f0-11e9-8390-b8ab1b42b9d0.png)